src/layers/feedforward.rs

```diff
@@ -0,0 +1,98 @@
+use candle_core::{Module, Tensor};
+use candle_nn::{linear, linear_no_bias, Linear, VarBuilder};
+
+use crate::error::Result;
+
+/// Feed-forward layers.
+
+/// Point-wise feed-forward layer (_Vaswani et al., 2017_).
```

Comment:
Does `rustdoc` support this syntax? Doesn't look like regular markdown to me.

Reply:
It does, makes the _Vaswani et al., 2017_ part italic.